lerna.json

```diff
@@ -3,7 +3,7 @@
         "packages/*"
     ],
     "npmClient": "yarn",
-    "version": "1.1.1",
+    "version": "2.0.0",
```

Comment:
Looks like this one is also bumping to 2.0.0 https://github.com/awslabs/synchro-charts/pull/123/files
We might want to send these out in the same release, so skip the version bump in both of these and send out a separate PR with the version bump containing both.